main.go

```diff
@@ -21,18 +21,23 @@ func main() {
 	r.HandleFunc("/expand", ExpandHandler).Methods("POST")
 	r.HandleFunc("/parser", ParserHandler).Methods("POST")
 
-	var host, certFile, keyFile string
-	var port int
-	flag.StringVar(&host, "listen-host", "0.0.0.0", "Listen host")
-	flag.IntVar(&port, "listen-port", 8080, "Listen port")
-	flag.StringVar(&certFile, "cert-file", "", "SSL Cert file")
-	flag.StringVar(&keyFile, "key-file", "", "SSL Key file")
-	listenSpec := fmt.Sprintf("%s:%d", host, port)
+	host := os.GetEnv("LISTEN_HOST")
+	if host == "" {
+		host = "0.0.0.0"
+	}
+	port := os.GetEnv("LISTEN_PORT")
+	if port == "" {
+		port = "5000"
```

Comment:
Is there any particular reason why you are using a default port of 5000 instead of 8080? 

I don't want to make a breaking change for others that are already using libpostal-rest.